src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -122,6 +125,13 @@ def cli_cosmosdb_create(cmd, client,
     if enable_public_network is not None:
         public_network_access = 'Enabled' if enable_public_network else 'Disabled'
 
+    system_assigned_identity = None
+    if assign_identity is not None:
+        if assign_identity == [] or (len(assign_identity) == 1 and assign_identity[0] == '[system]'):
+            system_assigned_identity = ManagedServiceIdentity(type='SystemAssigned')
+        else:
+            raise CLIError("Only '[system]' is supported right now for command argument '--assign-identity'.")
```

Comment:
please use error type in azclierror.py.
https://github.com/Azure/azure-cli/blob/3bff786b9e337aa35bb4cc64a09e5600a35663f8/src/azure-cli-core/azure/cli/core/azclierror.py#L152-L154

In addition, should such error be throw by server side if advanced feature is not supported?  If you only support system assigned identity, please refine your help message.